toolchest_client/tools/tool.py

```diff
@@ -85,7 +85,8 @@ def _prepare_inputs(self):
         if self.num_input_files < self.min_inputs:
             raise ValueError(f"Not enough input files submitted. "
                              f"Minimum is {self.min_inputs}, {self.num_input_files} found.")
-        if self.num_input_files > self.max_inputs:
+        # If max_inputs = None, this check is ignored.
```

Comment:
Nit: I don't think this comment is necessary, it's pretty clear by the `if self.max_inputs and`